onboarding/mentors.md

```diff
@@ -31,53 +71,83 @@ It's up to the mentoring pair to find what works best for them and to ensure the
 - Prior experience with our [technology stacks](/playbooks/technology-choices.md) (and others)
 - Other relevant past experience, particularly 'soft skills' that might seem orthogonal to writing code
-- Are they using the [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)? If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good things to do in between People Ops meetings, while working nearby each other or to pair on.
+- Are they using the
+  [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)?
+  If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good
+  things to do in between People Ops meetings, while working nearby each other or to pair on.
+
+**Your first week**
 
-### Your first week
-- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
+- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale
+  steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira,
+  github, notion, heroku, aws+hokusai, zeplin, ...)
 - Look for chances to pair or work alongside one another throughout the week.
-  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just setting it up, breaking and fixing a test.
+  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just
+    setting it up, breaking and fixing a test.
   - If they haven't set up hokusai yet, consider pairing on that.
-- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit things you talked about in the first meeting and look for areas that could have gone better. Consider amending relevant documents like this one.
+- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit
+  things you talked about in the first meeting and look for areas that could have gone better. Consider amending
+  relevant documents like this one.
 
-### Your first year
-- New hires usually begin with a sprint-by-sprint rotation through our teams, a whirlwind that can last upwards of 2 months. It's good to check in with your mentee during this time.
+**Your first year**
+
+- New hires usually begin with a sprint-by-sprint rotation through our teams, a whirlwind that can last upwards of
+  2 months. It's good to check in with your mentee during this time.
 - Consider reaching out to other mentors as well
```

Comment:
Let's add a bullet-point about the mentee joining their first team, which feels like a partial bookend to this!

Something like:
`- When the mentee joins their first full-time team, check in with them through the first few sprints. How is their ramp-up going? Are they feeling productive and safe within the team? Encourage the mentee to bring up these topics with their manager if you think it would help.`